BoxContentSDK/BoxContentSDK/Requests/BOXRecentItemsRequest.h

```diff
@@ -0,0 +1,41 @@
+//
+//  BOXRecentItemsRequest.h
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/19/16.
+//
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+
+@interface BOXRecentItemsRequest : BOXRequest
+
+/**
+ Number of records to fetch. 
+ - Minimum allowed value: 0
+ - Maximum allowed value: 1000
+ - Defaults to 100
```

Comment:
OK, having seen the implementation code, that appears to be the case. Awesome!
Should we make this comment say something like "Defaults to 100 at the time of writing" or something like that? :)